CDCgov/prime-reportstream
# 12723: Update Terraform to enable Java agent

This PR updates Terraform to enable the AppInsights Java agent in accordance with this documentation.

https://learn.microsoft.com/en-us/azure/azure-monitor/app/monitor-functions

Test Steps:
1. Once deployed to staging check to see if the telemetry data is coming through

## Changes
- Updates an Env var

### Testing
